wp-content/plugins/hurumap-ui/src/chart/EditChart.js

```diff
@@ -1,18 +1,39 @@
+import { useEffect, useState, useMemo } from "react";
 import { __ } from "@wordpress/i18n";
 import { Component, Fragment } from "@wordpress/element";
-import { PanelBody, Placeholder, SelectControl } from "@wordpress/components";
+import {
+  PanelBody,
+  Placeholder,
+  SelectControl,
+  TextControl
+} from "@wordpress/components";
 import { InspectorControls } from "@wordpress/editor";
 
-import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
+import { InsightChartContainer } from "@codeforafrica/hurumap-ui";
+import Chart from "../Chart";
 
-import { ChartContainer, BarChart } from "@codeforafrica/hurumap-ui";
+import sections from "../data/charts";
+import config from "../config";
 
-import Theme from "../Theme";
+import withRoot from "../withRoot";
+import { useQuery, useApolloClient } from "@apollo/react-hooks";
+import { GET_GEOGRAPHIES, buildDataCountQuery } from "../data/queries";
 
-export default class EditChart extends Component {
-  constructor(props) {
-    super(props);
-    this.data = Array(3)
+import { Grid } from "@material-ui/core";
+
+function EditChart({
+  clientId,
+  className,
+  attributes: { chartId: selectedChart, geoId: selectedGeo, chartWidth },
+  setAttributes,
+  posts,
+  selectedPostType,
+  postTypes,
+  ...props
+}) {
+  const client = useApolloClient();
+  const data = useState(
```

Comment:
A comment may be why we can't start with empty/null data

Reply:
This was for testing.
I will clean this up.